extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/QuarkusJaegerTracer.java

```diff
@@ -71,10 +76,15 @@ private Tracer tracer() {
         if (tracer == null) {
             synchronized (this) {
                 if (tracer == null) {
+                    ZipkinV2Reporter reporter = null;
```

Comment:
Have you built an application using the extension with this change into a native executable and it works as expected without error?

Reply:
I made what you requested and the results are:
without my changes:

- executable size: 43M
- RSS on startup: 18908
- RSS after first request: 22144

With my changes:
- executable size: 44M
- RSS on startup: 18960
- RSS after first request: 24796

It worked both for zipkin and jaeger instances.